tests/ui/specs/RangePicker/RangePicker-RTL-specs.js

```diff
@@ -0,0 +1,52 @@
+const Page = require('./RangePickerPage');
+const {extractValue} = require('./RangePicker-utils.js');
+
+describe('RangePicker RTL', function () {
+	beforeEach(async function () {
+		await Page.open('?locale=ur-PK');
+	});
+
+	describe('with negativeValues in RTL', function () {
+		const maxValueRangePicker = Page.components.rangePickerMaxValues;
+		const negativeRangePicker = Page.components.rangePickerWithNegativeValues;
+		const rangePickerDisabled = Page.components.rangePickerDisabled;
+
+		it('should decrement to negative number [QWTC-2148]', async function () {
+			// Step 5: Click on the Left arrow button until '-5' displays.
+			await negativeRangePicker.decrementer(negativeRangePicker.self).click();
+			await browser.pause(500);
+			const newValue = extractValue(negativeRangePicker);
+			// Step 5 Verify: The minus sign displays to the Left of the number '5'.
+			expect(await newValue).to.equal(-5);
```

Comment:
I think screenshot test will be needed. 
for `The *minus* sign displays to the Left of the number '*5*'.`

I'll add

Reply:
Thank you!